spec/support/macros.rb

```diff
@@ -77,5 +77,19 @@ def query_cache_enabled
         end
       end
     end
+
+    def with_default_i18n_configs
+      around do |ex|
+        I18n.locale = :en
```

Comment:
I'm not sure if it's necessary to do this before and after, but I felt like it was a more airtight solution. I also tried making a method that does this, but I wasn't able to access it from within the around block.